engine/src/audio/Listener.cpp

```diff
@@ -10,13 +10,13 @@
 namespace Audio {
 
     Listener::Listener() throw() :
-        cosAngleRange(-1,-1),
         position(0,0,0),
         atDirection(0,0,-1),
         upDirection(0,1,0),
         velocity(0,0,0),
         radius(1),
         gain(1),
+        cosAngleRange(-1,-1),
```

Comment:
any info on this change? Seems a little arbitrary

Reply:
The compiler complains when members are not initialized in the same order as they were declared.